README.md

```diff
@@ -398,3 +398,60 @@ change the UUID.
 
 You can't. This is because historical bookings still refer to that prison.
 Disable it instead (see above).
+
+## Smoke Tests
+
+Runs the happy path to book, confirm and cancel a visit.  If the app
+receives a an email in the format of
+`SMOKE_TEST_EMAIL_LOCAL_PART`@`SMOKE_TEST_EMAIL_DOMAIN`, it will not
+send that email to the prison, but will direct all mail the
+`SMOKE_TEST_EMAIL_HOST`.
+
+To start the smoke tests, set the environment variables and run
+`ruby ./smoke_test/smoke_test.rb`.
+
+The normal SMTP server details will need to be set and the app must be
+run in production mode for the smoke tests to work.
+
+##### Smoke Test Steps:
+  1. Starts the form
+  2. Fills in prisoner details page
+  3. Fills in visitor details page for one visitor
+  4. Selects three available days on the slot page
+  5. Submits the booking on the check your request page
+  6. Checks that the user received a booking receipt email
+  7. Checks that the prison received a booking request email
+  8. Visits the process booking page and confirms the booking
+  9. Checks that the prison received a copy of the confirmation email
+  10. Checks that the visitor received a confirmation email
+  11. Checks the status of the visit and cancels it
+  12. Checks that the prison receives a booking cancellation email
+
+##### Required environment variables for the rails app:
+```
+SMOKE_TEST_EMAIL_LOCAL_PART
+SMOKE_TEST_EMAIL_DOMAIN
+```
+
+##### Required environment variables for the smoke test:
+```
+SMOKE_TEST_EMAIL_LOCAL_PART
+SMOKE_TEST_EMAIL_DOMAIN
+SMOKE_TEST_EMAIL_PASSWORD
+SMOKE_TEST_APP_HOST
+SMOKE_TEST_EMAIL_HOST
+```
+### `SMOKE_TEST_EMAIL_LOCAL_PART`, `SMOKE_TEST_EMAIL_DOMAIN`
+
+These determine the email address that will trigger the smoke tests on
```

Comment:
`the email address that will trigger the smoke tests` … and to which smoke test emails will be sent?